launch/napoleon_navigation_plus_perception.launch

```diff
@@ -17,7 +17,9 @@
   
     <!--- Run AMCL -->
     <node pkg="amcl" type="amcl" name="amcl">
-        <remap from="scan" to="/$(arg robotName)/$(arg laser1_name)/scan"/>
+        <!-- <remap from="scan" to="/$(arg robotName)/$(arg laser1_name)/scan"/> -->
+        
+        <remap from="scan" to="/projected_scan_front"/> <!-- for BRSU ropod -->
```

Comment:
will this work in the robot?

Reply:
Yes, in our ropod we use lidar, hence we have to subscribe to this topic